includes/embeds/class-amp-playlist-embed-handler.php

```diff
@@ -57,7 +64,8 @@ class AMP_Playlist_Embed_Handler extends AMP_Base_Embed_Handler {
 	 * Registers the playlist shortcode.
 	 */
 	public function register_embed() {
-		add_shortcode( 'playlist', array( $this, 'shortcode' ) );
+		add_shortcode( self::SHORTCODE, array( $this, 'shortcode' ) );
+		add_action( 'wp_enqueue_scripts', array( $this, 'styling' ) );
```

Comment:
This use 'wp_enqueue_scripts,' instead of 'wp_playlist_script.' That enqueues too late.